docs/source/sctool/partials/sctool_progress.yaml

```diff
@@ -50,11 +50,13 @@ options:
 - name: run
   default_value: latest
   usage: |
-    Show progress of a particular run, see sctool info to get the `ID`s.
+    Show progress of a particular run, see sctool info to get the `IDs`.
```

Comment:
@mmatczuk Could you please update the code in `master` to generate this output? Sphinx complains if the `s` is after the "`" symbol.